erlang/pop_protocol/pop_protocol.erl

```diff
@@ -1,48 +1,78 @@
 -module(pop_protocol).
 
 %-export([add_new_transaction/2, add_new_block/2, generate_new_block/2]).
+%-import(blocktree, [add_new_transaction/2, add_new_block/2, generate_new_block/2]).
+-import(blocktree, [add_new_block/2, get_block_by_id/2]).
 
 -include("../blocktree/blocktree.hrl").
 -include_lib("stdlib/include/assert.hrl").
 
--record(protocol_data, {verifiers_lst, time_between_blocks, chain_id, tree_data}).
--record(consensus_block_data, {signature, signer_pub_key, timestamp}).
+-record(verifier_public_info, {index, public_key, network_data}).
+-record(protocol_data, {verifiers_arr, time_between_blocks, time_desync_margin, chain_id, tree_data}).
```

Comment:
rename to protocol_params?

Reply:
oh nvm, I see, verifiers_arrs and maybe tree data changes? The other ones seem static so maybe split out?